Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -709,86 +709,74 @@ def audit_specs
     end
 
     def audit_revision_and_version_scheme
+      return if @new_formula
+      return unless @strict
```

Comment:
We don't run `--strict` in the CI but this particular audit is most useful when done in the CI.

Reply:
Yeh, I had in my head we did run `--strict` in CI but you're right, we don't. I think this probably warrants another flag e.g. `--slow`, `--git` or even `--ci`.